reactDashboard/src/components/DataCharts/DataCharts.css

```diff
@@ -165,3 +165,7 @@ canvas {
 .chart-dropdown-container.row {
   margin: 0;
 }
+
+.jss1 {
```

Comment:
Again, wossat? :)

Reply:
I was trying to get rid of the gap between the date selector buttons and the chart - it is difficult to check because this only appears once it is running on live or staging that className appears to be attached to the offending element

Reply:
Yeah, its cos of https://github.com/Scottish-Tech-Army/STACovidDashboard/blob/5f509ddde6c149c0e9717a96b26c12103202757d/reactDashboard/src/components/DataCharts/DataCharts.css#L161
Thats not how Material UI styling works - the names are dynamically generated and are different between dev and prod builds. Deleting this should fix the issue https://github.com/Scottish-Tech-Army/STACovidDashboard/blob/5f509ddde6c149c0e9717a96b26c12103202757d/reactDashboard/src/components/DataCharts/DateRangeSlider.jsx#L11 (and removing that css block in DataCharts.css)

Reply:
removed block from css file and margin-top from inline styling